ui/app/configuration/directives/pnc-configuration-create-panel/pncConfigurationCreatePanel.js

```diff
@@ -0,0 +1,143 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+(function () {
+
+  var module = angular.module('pnc.configuration');
+
+  module.directive('pncConfigurationCreatePanel', [
+    function () {
+      
+      function PncConfigurationCreateController($log, $state, $filter, $scope, Notifications, 
+    		  EnvironmentDAO, ProjectDAO, ProductDAO, BuildConfigurationDAO, BuildConfigurationSetDAO) {
+
+    	// Selection of Product Versions.
+        $scope.productVersions = {
+    	  selected: []
```

Comment:
The indentation goes a bit wrong here, this line needs to be indented twice more and the indentation is broken from this line onwards.